src/BMPEncoder.ts

```diff
@@ -1,16 +1,31 @@
-'use strict';
+import { IOBuffer } from 'iobuffer';
 
-const { IOBuffer } = require('iobuffer');
+import { BITMAPV5HEADER } from './constants';
 
-const constants = require('./constants');
+export interface DataToEncode {
+  bitDepth: number;
+  height: number;
+  width: number;
+  data:
+    | (number | IOBuffer | ArrayBufferLike | ArrayBufferView | Buffer)
+    | undefined;
```

Comment:
I guess it's the same for `undefined`. What would we encode if there is no data?